HousingResidentInformationAPI.Tests/V1/Gateways/housingGatewayTests.cs

```diff
@@ -356,6 +356,31 @@ public void GetAllResidentsWithNoPhoneWithFirstAndLastNameQueryParameterReturnsM
             listOfPersons.Should().ContainEquivalentOf(domainEntity2);
         }
 
+        [Test]
+        public void GetAllResidentsWithNoContactLinkWithPostcodeQueryParameterReturnsMatchingResidents()
+        {
+            //Test records 1
```

Comment:
Are these comments neccessary, can it be removed, I personally feel as though without the test can be understandable that they are three different records

Reply:
I can remove them. It was just for me to follow what is happening in the code.

Reply:
I think it should be removed before merging to master

Reply:
Will do!